README.md

```diff
@@ -2,29 +2,45 @@
 
 NiftiMath rust project.
 
-This project provide a simple way to make mathematical operation on nifti images (the 
-same way mrcalc from [mrtrix3](https://github.com/MRtrix3/mrtrix3)).
+This project provides a simple way to do mathematical operation on nifti images the 
+same way mrcalc does in [mrtrix3](https://github.com/MRtrix3/mrtrix3)).
 
-The formulas are express in [reverse polish notation](https://en.wikipedia.org/wiki/Reverse_Polish_notation).
+The formulas are expressed in [reverse polish notation](https://en.wikipedia.org/wiki/Reverse_Polish_notation).
 
 ## Exemple
+Binary operation (Mul, Div, Sub, Add)
 ```
 niftimath 1.0 t1.nii.gz mul 3 6.1 add mul output.nii.gz
 ```
 This correspond to the following operation:
 ```
 output.nii.gz = (1.0 * t1.nii.gz) * (3 + 6.1)
+```
+
+Unary Operation (Abs, etc)
 ``` 
+niftimath 5.0 t1.nii.gz sub abs 10 mul output.nii.gz
+```
+This is equivalent to:
+```
+output.nii.gz = abs(5.0 - t1.nii.gz) * 10
+```
 
 ## Operations
 
+### Binary Operation
 - [X] Addition (add)
 - [X] Subtraction (sub)
 - [X] Multiplication (mul)
 - [X] Division (div)
-- [ ] Add possibility to save as a desired nifti type (u8, u16, f32, f64, ...)
+### Unary Opetation
 - [ ] Minimum (min), between images will be a min voxel wise.
 - [ ] Maximum (max), between images will be a max voxel wise.
```

Comment:
I don't understand this sentence. It's defined as an unary operation, but it's applying something "between images" (plural). It's either unary and returns a scalar, or binary and returns an image, no?

Reply:
Well I should have both, `{max,min}_reduce` that output a scalar given one image only. And a binary operation between images {max,min} that output an image of the {max,min} for each voxels in the images.